testing/test-helper.js

```diff
@@ -173,20 +167,26 @@ export class RequestBank {
    *   headers: JsonObject
    *   body: string
    * }
-   * @param id
+   * @param {number} id
    * @returns {Promise<JsonObject>}
    */
   static withdraw(id) {
-    if (id.indexOf('/') >= 0) {
-      throw new Error('ID "' + id + '" should not contain "/"');
-    }
-    const url = `${REQUEST_URL}/withdraw/${userAgent}-${id}/`;
+    const url = `${REQUEST_URL}/withdraw/${id}/`;
     return xhrServiceForTesting(window).fetchJson(url, {
       method: 'GET',
       ampCors: false,
       credentials: 'omit',
     }).then(res => res.json());
   }
+
+  static tearDown() {
```

Comment:
do we expect a response from server that tear down is successful? Is it possible that the `fetchJson` fails, and do we need timeout for that?

Reply:
yes we do need a response, to make avoid race condition between client and server.

i will try to see in case of server hang, mocha should still be able to fail instead of waiting forever.

will update here once confirmed

Reply:
tested: mocha has its own timeout on afterEach.

Reply:
Sounds good, thanks.